examples/offline_inference_with_profiler.py

```diff
@@ -15,19 +16,25 @@
 # Create a sampling params object.
 sampling_params = SamplingParams(temperature=0.8, top_p=0.95)
 
-# Create an LLM.
-llm = LLM(model="facebook/opt-125m", tensor_parallel_size=1)
+if __name__ == "__main__":
```

Comment:
Note that you shouldn't need the `if __name__ == "__main__":` once https://github.com/vllm-project/vllm/pull/11074 lands